spec/stat_generator_spec.rb

```diff
@@ -24,6 +24,66 @@
     end
   end
 
+  describe '#checks highest total score and lowest total score' do
```

Comment:
Generally this is the name of the method that you will be testing below and then the it block describes what it does that you're testing